packages/p2p/src/components/my-profile/my-profile.jsx

```diff
@@ -75,52 +95,94 @@ const MyProfile = observer(() => {
                         <Table.Row className='my-profile__stats'>
                             <div className='my-profile__stats-cell-separator' />
                             <Table.Cell className='my-profile__stats-cell'>
-                                <div className='my-profile__stats-cell-header'>{localize('Total orders')}</div>
-                                <div className='my-profile__stats-cell-info'>{total_orders_count || '-'}</div>
+                                <Text size={isMobile() ? 'xxxs' : 's'} color='less-prominent' lineheight='m' as='p'>
+                                    {localize('Total orders')}
+                                </Text>
+                                <Text color='prominent' weight='bold' lineHeight='l' as='p'>
+                                    {total_orders_count || '-'}
+                                </Text>
                             </Table.Cell>
                             <div className='my-profile__stats-cell-separator' />
-                            <Table.Cell className='my-profile__stats-cell'>
-                                <div className='my-profile__stats-cell-header'>
-                                    {localize('Buy ({{currency}})', { currency: general_store.client.currency })}
-                                </div>
-                                <div className='my-profile__stats-cell-info'>{buy_orders_count || '-'}</div>
-                            </Table.Cell>
+                            {isMobile() ? (
+                                <Table.Cell className='my-profile__stats-cell'>
+                                    <Text size={isMobile() ? 'xxxs' : 's'} color='less-prominent' lineheight='m' as='p'>
+                                        {localize('Buy / Sell ({{currency}})', {
+                                            currency: general_store.client.currency,
+                                        })}
+                                    </Text>
+                                    <Text color='prominent' weight='bold' lineHeight='l' as='p'>
+                                        {buy_orders_count || '-'}/{sell_orders_count || '-'}
+                                    </Text>
+                                </Table.Cell>
+                            ) : (
+                                <>
+                                    <Table.Cell className='my-profile__stats-cell'>
+                                        <Text
+                                            size={isMobile() ? 'xxxs' : 's'}
+                                            color='less-prominent'
+                                            lineheight='m'
```

Comment:
Can you check the Text component? I believe it's `lineHeight`.

Reply:
Please change all `lineheight` to `lineHeight`, you have used both right now. 😅

Reply:
updated